content/examples/_index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: "QCArchive Examples"
+date: 2019-04-17T12:35:54+06:00
+description: "Stories and news about the QCArchive"
```

Comment:
description and pageDescription need to be updated (or maybe deleted?)

Reply:
Isn't used, but updated anyway.